packages/excalidraw/components/App.tsx

```diff
@@ -8865,6 +8865,15 @@ class App extends React.Component<AppProps, AppState> {
               }));
 
               this.scene.replaceAllElements(elementsWithIndices);
+              elementsWithIndices.forEach((element) => {
+                if (
+                  isBindableElement(element) &&
+                  element.boundElements?.some((other) => other.type === "arrow")
+                ) {
+                  updateBoundElements(element, this.scene);
```

Comment:
this updates unrelated elements, which even if it ends up being stable (in the rewrite), we shouldn't be doing, because of forward compat (e.g. we change routing or something, and we don't want to update it for elements you do not modify explicitly).

Reply:
Got it, it iterates all the scene not just the dragged bindables.